src/components/_overlays/Modal/Modal.styles.ts

```diff
@@ -23,5 +27,6 @@ export const ModalContent = styled.div`
   max-height: 90vh;
   max-width: 90vw;
   overflow: hidden;
-  box-shadow: 0 8px 8px rgba(0 0 0 / 0.12), 0 24px 40px rgba(0 0 0 / 0.16);
+  ${({ noBoxShadow }) =>
+    !noBoxShadow && `box-shadow: ${cVar('effectElevation24Layer2')}, ${cVar('effectElevation24Layer1')}`};
```

Comment:
Let's move the styles to a separate `css` `const` so we don't use styles from strings